### PR TITLE
fix: add missing resources for appbarbutton

### PR DESCRIPTION
**Pull request by kazo0**
fixes https://github.com/unoplatform/private/issues/525

Introduce AppBarButtonHeight/Width theme resources

### CODE REVIEW

File: src/library/Uno.Material/Styles/Controls/v2/AppBarButton.xaml [hunk 1]
```diff
@@ -8,6 +8,17 @@
 					xmlns:ios="http://uno.ui/ios"
 					xmlns:uuc="using:Uno.UI.Controls"
 					mc:Ignorable="android ios">
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Light">
+			<x:Double x:Key="AppBarButtonHeight">64</x:Double>
```
Comment (agneszitte) [inline src/library/Uno.Material/Styles/Controls/v2/AppBarButton.xaml:13]: @kazo0 I have a little doubt, are we not missing documentation for AppBarButton like we have [here for Button](https://github.com/unoplatform/Uno.Themes/blob/master/doc/styles/Button.md) ?

**Review by agneszitte — APPROVED** [review]

**Comment by agneszitte** [comment]
@kazo0 backport to release/stable5.2 needed?

File: src/library/Uno.Material/Styles/Controls/v2/AppBarButton.xaml (hunk 1 above)
Comment (kazo0) [reply src/library/Uno.Material/Styles/Controls/v2/AppBarButton.xaml:13]: yeah it looks like we don't have a docs page for the styles/resources for AppBarButton. It's sort of a special case as these are used specifically for the NavigationBar. But I will add a docs page to the PR

**Comment by kazo0** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**